tests/pr_test.go

```diff
@@ -134,3 +136,50 @@ func TestRunBasicExampleWithFlavor(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunStandardSolution(t *testing.T) {
```

Comment:
We can't just add two new tests to every PR run. 

Consider dropping TestRunBasicExampleWithFlavor since the 'flavor' is used in TestRunFSCloudExample anyway.
Consider dropping TestRunUpgradeCompleteExample although that might be a gap in that PR would only cover Isolated. So maybe drop TestRunFSCloudExample and ensure the DA is testing on isolated hosting.

Reply:
Sure @shemau , I understand your point about  not adding two new tests to every PR run. Can you please suggest is my understanding correct here - 

Is this about **only** dropping `TestRunFSCloudExample` i.e. moving this to `other_tests` and keeping other tests i.e. DA tests and `TestRunBasicExampleWithFlavor` as it is in `pr_tests`?